tests/integration/targets/ndo_mcp_global_policy/tasks/main.yml

```diff
@@ -0,0 +1,300 @@
+
+    - name: Delete a MCP Global Policy with name
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *delete_mcp_global_policy
+      register: nm_delete_mcp_global_policy
+
+    - name: Delete MCP Global Policy with name again
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *delete_mcp_global_policy
+      register: nm_delete_mcp_global_policy_again
+
+    - name: Assert that the MCP Global Policy was deleted with name
+      ansible.builtin.assert:
+        that:
+          - cm_delete_mcp_global_policy is changed
+          - nm_delete_mcp_global_policy is changed
+          - cm_delete_mcp_global_policy.current == nm_delete_mcp_global_policy.current == {}
+          - cm_delete_mcp_global_policy.previous.name == nm_delete_mcp_global_policy.previous.name == "ansible_mcp_global_policy_new"
+          - cm_delete_mcp_global_policy.previous.key == nm_delete_mcp_global_policy.previous.key == "cisco"
+          - cm_delete_mcp_global_policy.previous.description == nm_delete_mcp_global_policy.previous.description == "A Global MCP Policy"
+          - nm_delete_mcp_global_policy.previous.uuid is defined
+          - nm_delete_mcp_global_policy_again is not changed
+          - nm_delete_mcp_global_policy_again.previous == nm_delete_mcp_global_policy_again.current == {}
+
+    - name: Create the MCP Global Policy
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *add_mcp_global_policy
+        name: ansible_mcp_global_policy_again
+        description: An MCP Global policy to delete with UUID
+      register: new_mcp_global_policy
+
+    - name: Delete the MCP Global Policy with incorrect UUID
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *mso_info
+        template: ansible_fabric_policy_template
+        uuid: does-not-exist
+        state: absent
+      ignore_errors: true
+      register: no_uuid_delete_mcp_global_policy
+
+    - name: Delete the MCP Global Policy with UUID
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *mso_info
+        template: ansible_fabric_policy_template
+        uuid: '{{ new_mcp_global_policy.current.uuid }}'
+        state: absent
+      register: nm_delete_mcp_global_policy_uuid
+
+    - name: Assert that the MCP Global Policy was deleted with UUID
+      ansible.builtin.assert:
+        that:
+          - nm_delete_mcp_global_policy_uuid is changed
+          - nm_delete_mcp_global_policy_uuid.previous.uuid is defined
+          - nm_delete_mcp_global_policy_uuid.previous.name == "ansible_mcp_global_policy_again"
+          - nm_delete_mcp_global_policy_uuid.previous.description == "An MCP Global policy to delete with UUID"
+          - nm_delete_mcp_global_policy_uuid.current == {}
+
+    # Errors and no policy found
+    - name: Query the MCP Global Policy in the template when its deleted
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *query_mcp_global_policy
+      register: query_none
+
+    - name: Create an MCP Global Policy without a name
```

Comment:
Is this test input correct, because I would argue that a create with UUID is not possible because the policy is not created yet. So uuid is not needed to be provided?